x-pack/plugin/esql/src/main/antlr/parser/Expression.g4

```diff
@@ -49,7 +49,8 @@ primaryExpression
     ;
 
 functionExpression
-    : functionName LP (ASTERISK | (booleanExpression (COMMA booleanExpression)* (COMMA mapExpression)?))? RP
+    : functionName LP (ASTERISK | (booleanExpression (COMMA booleanExpression)* (COMMA mapExpression)?))? RP  #functionStandard
+    | (FIRST | LAST) LP value=booleanExpression BY by=booleanExpression RP                                             #functionFirstLast
```

Comment:
Why value and by are boolean expressions? I would expect something about column identifier.

Also I am a bit worried about making a custom syntax for each function.
I understand the workaround for FIRST and LAST, however I am not sure if we should also have `BY` in it opposed to a simpler coma separated list of parameters.
I am imagining something like:
```
name=(functionName | FIRST | LAST) LP (ASTERISK | (booleanExpression (COMMA booleanExpression)* (COMMA mapExpression)?))? RP
```